visualization-project-laptop-defects/visual.py

```diff
@@ -0,0 +1,41 @@
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+import datetime as dt
+
+data = pd.read_excel("Data_Analysis_Project.xlsx", 'Data')
+
+data['Month ID(YYYYMM)']=pd.to_datetime(data['Month ID(YYYYMM)'], format="%Y%m")
+
+sns.barplot(x= 'Month ID(YYYYMM)', y= 'Defects', data=data)
```

Comment:
Normally we would graph time series data using a line plot. This looks nice though so changing it is optional.